scala/src/main/scala/com/concord/wordcounter/WordCounter.scala

```diff
@@ -0,0 +1,54 @@
+package com.concord.wordcounter
+
+import java.util
+
+import com.concord._
+import com.concord.swift._
+
+class WordCounter extends Computation {
+
+  private val histogram: util.HashMap[String, Integer] = new util.HashMap()
```

Comment:
I would even alias it as ` MutableHashMap ` to keep in sync w/ scala collection naming. 

```
import java.util.{ HashMap => MutableHashMap }
```